client/src/app/+login/login.component.scss

```diff
@@ -17,6 +17,33 @@ input[type=submit] {
   @include orange-button;
 }
 
+.modal-body {
+  text-align: left;
+
+  .forgot-password-instructions {
+    margin-bottom: 20px;
+  }
+
+  #forgot-password-email {
+    width: 100%;
+
+    @media screen and (min-width: #{map-get($container-max-widths, sm)}) {
+      width: 350px;
```

Comment:
You redefine the width that was set at line 9, because you have overriden it line 28. Please reverse your media-query so that you only override it to `width: 100%` on `max-width: #{map-get($container-max-widths, sm)}`.